torchmdnet/models/output_modules.py

```diff
@@ -11,9 +12,10 @@
 
 
 class OutputModel(nn.Module, metaclass=ABCMeta):
-    def __init__(self, allow_prior_model):
+    def __init__(self, allow_prior_model, reduce_op):
```

Comment:
Old one had a default for reduce_op="add". Not sure if it's still needed though. Do all outputmodels explicitly define their reduction operation?